### PR TITLE
Update Trigger Flow image and add it to the docs.

### PR DESCRIPTION
**Pull request by khrm**
# Changes
Update Trigger Flow image and add it to the docs/README.md

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes
```release-note
NONE
```

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ the functionality of Tekton Pipelines to support events:
 
    **Note:** `Interceptors` are currently part of `EventListeners` but are being converted to a standalond CRD in [PR 960](https://github.com/tektoncd/triggers/pull/960).
```
Comment (dibyom) [inline docs/README.md:49]: could we remove this too?

**Review by dibyom — COMMENTED** [review]
The diagram shows that the event hits the ClusterInterceptor before hitting the EL Pod. Is it possible to show that the event is hitting the EL pod first? And then the EL Pod is talking to the ClusterInterceptors before the Bindings/Template step?

**Comment by khrm** [comment]
I updated the image. Do we need to draw it more explicitly, @dibyom ?

**Comment by sm43** [comment]
image

should we change the font color for `Event` and other terms not on the white section?

**Comment by khrm** [comment]
@sm43 I changed the background. Please verify that they are legible in your environment.

**Comment by sm43** [comment]
> @sm43 I changed the background. Please verify that they are legible in your environment.

lgtm Thank you.

**Comment by dibyom** [comment]
> I updated the image. Is this fine or do we need to draw the flow more explicitly, @dibyom ?

This looks good

/approve

**Pull request merged**